libraries/AP_Baro/AP_Baro_BMP388.cpp

```diff
@@ -107,6 +109,9 @@ bool AP_Baro_BMP388::init()
         return false;
     }
 
+    dev->write_register(BMP388_REG_CMD, BMP388_SOFT_RESET, true);
```

Comment:
the true is wrong, this is not a checked register, this will reset your baro every 20 loops

Reply:
I removed this line, so now the PR only adds the new sensor to the navigator.
I tested it here and it worked fine on my board without the soft reset. If the issue comes up again, I'll open a new PR for it.